cmd/mdatagen/metadata-schema.yaml

```diff
@@ -31,7 +42,7 @@ attributes:
     # Required: attribute value type.
     type: <string|int|double|bool|bytes|slice|map>
 
-# Required: map of metric names with the key being the metric name and value
```

Comment:
It looks like the metadata generator started as something specific to metrics but it doesn't look like it is a required field anymore.

Reply:
true